Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.7.2"
 edition = "2018"
 authors = [
     "Sven-Hendrik Haase <svenstaro@gmail.com>",
-    "Alexander Dmitriev <alexander.dmitriev2580@gmail.com>"
+    "Alexander Dmitriev <alexander.dmitriev2580@gmail.com>",
```

Comment:
Would you like to add yourself as author?

Reply:
Ah, sure. Is everything else alright? If so I can just merge it myself when I add it in.

Reply:
True, you can do that.